src/grass-dev/osgeo4w/package.sh

```diff
@@ -0,0 +1,111 @@
+export P=grass-dev
+export PACKAGE_POSTFIX="-dev"
+export V=8.0.dev
+export B=next
+export MAINTAINER=JuergenFischer
+export BUILDDEPENDS="gdal-devel pdal-devel proj-devel geos-devel netcdf-devel libjpeg-devel libpng-devel libpq-devel libtiff-devel sqlite3-devel zstd-devel python3-ply python3-core python3-six python3-pywin32 python3-wxpython wxwidgets-devel"
+
+branch=main
+
+if [ "$CI" ] ; then
+cd "$OSGEO4W_REP"
```

Comment:
OSGEO4W_REP is supposed to be the directory where the package repository is (which is not neccessarilly the the base repository)

Reply:
OK, I just use that as entry point in my github action where OSGEO4W_REP is set. Can use another variable name here.

Reply:
Variable renamed in 3e8c9a8